lib/broccoli/translation-reducer/utils/wrap-with-namespace-if-needed.js

```diff
@@ -28,7 +28,7 @@ function wrapWithNamespaceIfNeeded(object, filepath, inputPath, addonNames) {
     for (let addon of normalizedAddonNames) {
       let addonPrefix = `${path.sep}${enums.addonNamespace}${path.sep}${addon}`;
 
-      if (dirname.startsWith(addonPrefix)) {
+      if (`${dirname}/`.startsWith(`${addonPrefix}/`)) {
```

Comment:
When I tried to set up CI to run the Node tests on Windows, I found that the `/` causes the tests to fail. I'll create a pull request to replace the hardcoded character with `path.sep` (line 29 shows an example).